ckan/tests/test_coding_standards.py

```diff
@@ -587,6 +587,7 @@ def find_unprefixed_string_literals(filename):
     u'ckanext/datastore/controller.py',
     u'ckanext/datastore/db.py',
     u'ckanext/datastore/backend/postgres.py',
+    u'ckanext/datastore/backend/example-sqlite.py',
```

Comment:
This list is for existing source files that have not yet been converted to the new standard. Please do not add new files to the list, instead make sure that they follow our [rules for Unicode handling](http://docs.ckan.org/en/latest/contributing/unicode.html) so that they pass the tests.